roles/robot.xml

```diff
@@ -0,0 +1,682 @@
+        <inherit>mod/lesson:addinstance</inherit>
+        <inherit>mod/lesson:edit</inherit>
+        <inherit>mod/lesson:manage</inherit>
+        <inherit>mod/lti:addcoursetool</inherit>
+        <inherit>mod/lti:addinstance</inherit>
+        <inherit>mod/lti:grade</inherit>
+        <inherit>mod/lti:manage</inherit>
+        <inherit>mod/lti:requesttooladd</inherit>
+        <inherit>mod/lti:view</inherit>
+        <inherit>mod/mylabmastering:addinstance</inherit>
+        <inherit>mod/oublog:addinstance</inherit>
+        <inherit>mod/oublog:audit</inherit>
+        <inherit>mod/oublog:comment</inherit>
+        <inherit>mod/oublog:contributepersonal</inherit>
+        <inherit>mod/oublog:exportownpost</inherit>
+        <inherit>mod/oublog:exportpost</inherit>
+        <inherit>mod/oublog:grade</inherit>
+        <inherit>mod/oublog:managecomments</inherit>
+        <inherit>mod/oublog:managelinks</inherit>
+        <inherit>mod/oublog:manageposts</inherit>
+        <inherit>mod/oublog:post</inherit>
+        <inherit>mod/oublog:rate</inherit>
+        <inherit>mod/oublog:view</inherit>
+        <inherit>mod/oublog:viewallratings</inherit>
+        <inherit>mod/oublog:viewanyrating</inherit>
+        <inherit>mod/oublog:viewindividual</inherit>
+        <inherit>mod/oublog:viewparticipation</inherit>
+        <inherit>mod/oublog:viewpersonal</inherit>
+        <inherit>mod/oublog:viewprivate</inherit>
+        <inherit>mod/oublog:viewrating</inherit>
+        <inherit>mod/page:addinstance</inherit>
+        <inherit>mod/page:view</inherit>
+        <inherit>mod/quiz:addinstance</inherit>
+        <inherit>mod/quiz:attempt</inherit>
+        <inherit>mod/quiz:deleteattempts</inherit>
+        <inherit>mod/quiz:emailconfirmsubmission</inherit>
+        <inherit>mod/quiz:emailnotifysubmission</inherit>
+        <inherit>mod/quiz:emailwarnoverdue</inherit>
+        <inherit>mod/quiz:grade</inherit>
+        <inherit>mod/quiz:ignoretimelimits</inherit>
+        <inherit>mod/quiz:manage</inherit>
+        <inherit>mod/quiz:manageoverrides</inherit>
+        <inherit>mod/quiz:preview</inherit>
+        <inherit>mod/quiz:regrade</inherit>
+        <inherit>mod/quiz:reviewmyattempts</inherit>
+        <inherit>mod/quiz:viewreports</inherit>
+        <inherit>mod/resource:addinstance</inherit>
+        <inherit>mod/scorm:addinstance</inherit>
+        <inherit>mod/scorm:deleteownresponses</inherit>
+        <inherit>mod/scorm:deleteresponses</inherit>
+        <inherit>mod/scorm:savetrack</inherit>
+        <inherit>mod/scorm:skipview</inherit>
+        <inherit>mod/scorm:viewreport</inherit>
+        <inherit>mod/scorm:viewscores</inherit>
+        <inherit>mod/survey:addinstance</inherit>
+        <inherit>mod/survey:download</inherit>
+        <inherit>mod/survey:participate</inherit>
+        <inherit>mod/surveypro:accessadvanceditems</inherit>
+        <inherit>mod/surveypro:accessownreports</inherit>
```

Comment:
survey pro and turnitin capabiliyes - please export this cleanly from a vanilla moodle